### PR TITLE
feat: support mean v2

### PR DESCRIPTION
**Pull request by 0xsambugs**
Hey Llamas!

We recently launched v2 at [Mean Finance](https://defillama.com/protocol/mean-finance). It's currently live on Polygon and Optimism, and we would love for v2 to be tracked on Defi Llama. This PR _should_ contain all necessary changes.

Also, if possible, we'd like to make a few changes to the displayed information:
- New description: "Mean Finance is the state-of-the-art DCA protocol. Our protocol enables users to Dollar Cost Average (DCA) any ERC20 into any ERC20 with their preferred period frequency by leveraging both Chainlink and Uniswap V3 TWAP oracles."
- Category: "Services" (we never did yield to begin with 😅 )
- Audits: We have [two](https://github.com/Mean-Finance/dca-v2-core/tree/main/audits) now! 
- Oracle used: Chainlink & TWAP

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/meanfinance/index.js exports TVL: 
        
 
 
    optimism                  53.90 k
    polygon                   142.38 k
    ethereum                  47.59 k
    
    total                    243.88 k

File: projects/meanfinance/index.js [hunk 1]
```diff
@@ -1,5 +1,56 @@
...
+      id
+    }
+  }`
+;
+
+const getTokensInChain = async (chain) => {
+  try {
+    const result = await request(GRAPH_URLS[chain], query);
+    return result.tokens.map(({ id }) => id)
+  } catch (e) {
```
Comment (g1nt0ki) [inline projects/meanfinance/index.js:27]: can you remove the try catch block, we would like to know/notify you when it fails :)

File: projects/meanfinance/index.js [hunk 2]
```diff
@@ -1,5 +1,56 @@
...
+  const tokens = await getTokensInChain(chain)
+  const hubAddress = DCA_HUB_ADDRESSES[chain]
+  for (const token of tokens) {
+    const balance = await sdk.api.erc20.balanceOf({
+      target: token,
+      owner: hubAddress,
+      block,
+      chain
+    })
+    sdk.util.sumSingleBalance(balances, `${chain}:${token}`, balance.output);
```
Comment (g1nt0ki) [inline projects/meanfinance/index.js:50]: [Suggestion]

`${chain}:${token}` this probably would work, but I can recommend fetching chain's transform function and using it

    const { getChainTransform } = require("../helper/portedTokens")

File: projects/meanfinance/index.js (hunk 1 above)
Comment (0xsambugs) [reply projects/meanfinance/index.js]: Done! 😄

**Review by g1nt0ki — APPROVED** [review]

**Comment by llamatester** [comment]
The adapter at projects/meanfinance/index.js exports TVL: 
        
 
 
    optimism                  54.94 k
    polygon                   142.97 k
    ethereum                  48.10 k
    
    total                    246.01 k

**Comment by 0xngmi** [comment]
@realdealshaman can you update the info and refill data?

**Pull request merged**